fuzz/fuzz_package_resolved.py

```diff
@@ -0,0 +1,78 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import json
+import sys
+import tempfile
+from pathlib import Path
+
+import atheris
+import atheris_libprotobuf_mutator
+from google.protobuf.json_format import MessageToDict
+
+import fuzz.generated.package_resolved_pb2 as package_resolved_pb2
+from cve_bin_tool.cvedb import CVEDB
+from cve_bin_tool.log import LOGGER
+
+with atheris.instrument_imports():
+    from cve_bin_tool.parsers.swift import SwiftParser
+
+cve_db = CVEDB()
+logger = LOGGER.getChild("Fuzz")
+
+
+def PackageResolvedBuilder(data):
+    # Convert the Protobuf message to a dictionary
```

Comment:
Same comment as the other one: let's change this to a docstring. (and I wouldn't say no if you wanted to add a docstring to TestParseData too.)